src/devtran.cpp

```diff
@@ -243,13 +244,13 @@ Rcpp::List DEVTRAN(const Rcpp::List parin,
     // Vector of vectors
     // Outer vector: length = number of designs
     // Inner vector: length = number of times in that design
-    std::vector<std::vector<rec_ptr> > designs;
+    recstack designs;
```

Comment:
`recstack` is a vector of `reclist`. This is another place where I wasn't using the typedef.